2020-06-04 18:33:00 ERROR (MainThread) [homeassistant.components.ipp] Error fetching ipp data: Invalid response from API: Timeout occurred while connecting to IPP server.
2020-06-04 18:52:40 ERROR (MainThread) [homeassistant.core] Error doing job: Fatal read error on socket transport
Traceback (most recent call last):
  File "/usr/lib/python3.7/asyncio/selector_events.py", line 801, in _read_ready__data_received
    data = self._sock.recv(self.max_size)
TimeoutError: [Errno 110] Connection timed out
2020-06-04 19:25:27 ERROR (MainThread) [homeassistant.core] Error doing job: Fatal read error on socket transport
Traceback (most recent call last):
  File "/usr/lib/python3.7/asyncio/selector_events.py", line 801, in _read_ready__data_received
    data = self._sock.recv(self.max_size)
TimeoutError: [Errno 110] Connection timed out
2020-06-04 20:36:53 ERROR (MainThread) [homeassistant.components.ipp] Error fetching ipp data: Invalid response from API: Timeout occurred while connecting to IPP server.
2020-06-04 22:01:43 ERROR (MainThread) [homeassistant.components.ipp] Error fetching ipp data: Invalid response from API: Timeout occurred while connecting to IPP server.
2020-06-05 06:47:26 ERROR (MainThread) [frontend.js.latest.202004272] http://10.10.6.15:8123/frontend_latest/chunk.9cedb8f1a54a4b78de8b.js:2901:346 Uncaught TypeError: Cannot convert undefined or null to object
2020-06-05 06:50:38 ERROR (MainThread) [frontend.js.latest.202004272] http://10.10.6.15:8123/frontend_latest/chunk.9cedb8f1a54a4b78de8b.js:2901:346 Uncaught TypeError: Cannot convert undefined or null to object
2020-06-05 06:57:35 ERROR (MainThread) [homeassistant.components.ipp] Error fetching ipp data: Invalid response from API: Timeout occurred while connecting to IPP server.
2020-06-05 09:22:16 WARNING (MainThread) [homeassistant.components.mqtt.binary_sensor] No matching payload found for entity: Driveway Alert with state topic: rtl_433/raspberrypi/devices/DrivewayAlert/rows/0/data. Payload: 8, with value template None
2020-06-05 12:55:03 ERROR (MainThread) [homeassistant.components.ipp] Error fetching ipp data: Invalid response from API: Timeout occurred while connecting to IPP server.
2020-06-05 13:07:58 ERROR (MainThread) [homeassistant.components.ipp] Error fetching ipp data: Invalid response from API: Timeout occurred while connecting to IPP server.
2020-06-05 13:22:02 ERROR (MainThread) [homeassistant.components.ipp] Error fetching ipp data: Invalid response from API: Timeout occurred while connecting to IPP server.
2020-06-05 13:29:23 ERROR (MainThread) [homeassistant.components.ipp] Error fetching ipp data: Invalid response from API: Timeout occurred while connecting to IPP server.
2020-06-05 13:38:41 ERROR (MainThread) [homeassistant.components.ipp] Error fetching ipp data: Invalid response from API: Timeout occurred while connecting to IPP server.
2020-06-05 13:51:28 ERROR (MainThread) [homeassistant.components.ipp] Error fetching ipp data: Invalid response from API: Timeout occurred while connecting to IPP server.
2020-06-05 14:01:28 ERROR (MainThread) [homeassistant.components.ipp] Error fetching ipp data: Invalid response from API: Timeout occurred while connecting to IPP server.
2020-06-05 14:20:56 ERROR (MainThread) [homeassistant.components.ipp] Error fetching ipp data: Invalid response from API: Timeout occurred while connecting to IPP server.
2020-06-05 14:29:50 WARNING (MainThread) [homeassistant.components.mqtt.binary_sensor] No matching payload found for entity: Driveway Alert with state topic: rtl_433/raspberrypi/devices/DrivewayAlert/rows/0/data. Payload: e, with value template None
2020-06-05 14:30:57 ERROR (MainThread) [homeassistant.components.ipp] Error fetching ipp data: Invalid response from API: Timeout occurred while connecting to IPP server.
2020-06-05 14:35:35 ERROR (MainThread) [homeassistant.components.ipp] Error fetching ipp data: Invalid response from API: Timeout occurred while connecting to IPP server.
2020-06-05 14:56:51 ERROR (MainThread) [homeassistant.components.ipp] Error fetching ipp data: Invalid response from API: Timeout occurred while connecting to IPP server.
2020-06-05 15:09:24 ERROR (MainThread) [homeassistant.components.ipp] Error fetching ipp data: Invalid response from API: Timeout occurred while connecting to IPP server.
2020-06-05 15:12:33 ERROR (MainThread) [homeassistant.components.ipp] Error fetching ipp data: Invalid response from API: Timeout occurred while connecting to IPP server.
2020-06-05 15:18:44 ERROR (MainThread) [homeassistant.components.ipp] Error fetching ipp data: Invalid response from API: Timeout occurred while connecting to IPP server.
2020-06-05 18:32:44 ERROR (MainThread) [homeassistant.components.ipp] Error fetching ipp data: Invalid response from API: Timeout occurred while connecting to IPP server.
2020-06-05 21:05:32 ERROR (MainThread) [homeassistant.core] Error doing job: Fatal read error on socket transport
Traceback (most recent call last):
  File "/usr/lib/python3.7/asyncio/selector_events.py", line 801, in _read_ready__data_received
    data = self._sock.recv(self.max_size)
TimeoutError: [Errno 110] Connection timed out
2020-06-05 21:05:32 ERROR (MainThread) [homeassistant.core] Error doing job: Fatal read error on socket transport
Traceback (most recent call last):
  File "/usr/lib/python3.7/asyncio/selector_events.py", line 801, in _read_ready__data_received
    data = self._sock.recv(self.max_size)
TimeoutError: [Errno 110] Connection timed out
2020-06-05 21:05:32 ERROR (MainThread) [homeassistant.core] Error doing job: Fatal read error on socket transport
Traceback (most recent call last):
  File "/usr/lib/python3.7/asyncio/selector_events.py", line 801, in _read_ready__data_received
    data = self._sock.recv(self.max_size)
TimeoutError: [Errno 110] Connection timed out
2020-06-05 21:05:32 ERROR (MainThread) [homeassistant.core] Error doing job: Fatal read error on socket transport
Traceback (most recent call last):
  File "/usr/lib/python3.7/asyncio/selector_events.py", line 801, in _read_ready__data_received
    data = self._sock.recv(self.max_size)
TimeoutError: [Errno 110] Connection timed out
2020-06-05 22:11:04 ERROR (MainThread) [homeassistant.core] Error doing job: Fatal read error on socket transport
Traceback (most recent call last):
  File "/usr/lib/python3.7/asyncio/selector_events.py", line 801, in _read_ready__data_received
    data = self._sock.recv(self.max_size)
TimeoutError: [Errno 110] Connection timed out
2020-06-06 01:13:58 ERROR (MainThread) [homeassistant.components.ipp] Error fetching ipp data: Invalid response from API: Timeout occurred while connecting to IPP server.
2020-06-06 02:31:16 ERROR (MainThread) [homeassistant.components.ipp] Error fetching ipp data: Invalid response from API: Timeout occurred while connecting to IPP server.
2020-06-06 08:16:35 ERROR (MainThread) [homeassistant.components.ipp] Error fetching ipp data: Invalid response from API: Timeout occurred while connecting to IPP server.
2020-06-06 08:57:11 ERROR (MainThread) [homeassistant.components.ipp] Error fetching ipp data: Invalid response from API: Timeout occurred while connecting to IPP server.
2020-06-06 09:09:24 ERROR (MainThread) [homeassistant.components.ipp] Error fetching ipp data: Invalid response from API: Timeout occurred while connecting to IPP server.
2020-06-06 09:22:24 WARNING (MainThread) [homeassistant.components.http.ban] Login attempt or request with invalid authentication from 10.10.6.148
2020-06-06 09:22:24 WARNING (MainThread) [homeassistant.components.http.ban] Login attempt or request with invalid authentication from 10.10.6.148
2020-06-06 09:22:24 WARNING (MainThread) [homeassistant.components.http.ban] Login attempt or request with invalid authentication from 10.10.6.148
2020-06-06 09:22:24 WARNING (MainThread) [homeassistant.components.http.ban] Login attempt or request with invalid authentication from 10.10.6.148
2020-06-06 09:22:24 WARNING (MainThread) [homeassistant.components.http.ban] Login attempt or request with invalid authentication from 10.10.6.148
2020-06-06 09:22:24 WARNING (MainThread) [homeassistant.components.http.ban] Login attempt or request with invalid authentication from 10.10.6.148
2020-06-06 09:22:24 WARNING (MainThread) [homeassistant.components.http.ban] Login attempt or request with invalid authentication from 10.10.6.148
2020-06-06 09:22:24 WARNING (MainThread) [homeassistant.components.http.ban] Login attempt or request with invalid authentication from 10.10.6.148
2020-06-06 09:22:24 WARNING (MainThread) [homeassistant.components.http.ban] Login attempt or request with invalid authentication from 10.10.6.148
2020-06-06 09:22:24 WARNING (MainThread) [homeassistant.components.http.ban] Login attempt or request with invalid authentication from 10.10.6.148
2020-06-06 09:22:24 WARNING (MainThread) [homeassistant.components.http.ban] Login attempt or request with invalid authentication from 10.10.6.148
2020-06-06 09:22:24 WARNING (MainThread) [homeassistant.components.http.ban] Login attempt or request with invalid authentication from 10.10.6.148
2020-06-06 09:22:24 WARNING (MainThread) [homeassistant.components.http.ban] Login attempt or request with invalid authentication from 10.10.6.148
2020-06-06 09:22:24 WARNING (MainThread) [homeassistant.components.http.ban] Login attempt or request with invalid authentication from 10.10.6.148
2020-06-06 10:25:22 ERROR (MainThread) [homeassistant.components.ipp] Error fetching ipp data: Invalid response from API: Timeout occurred while connecting to IPP server.
2020-06-06 10:31:41 ERROR (MainThread) [homeassistant.components.ipp] Error fetching ipp data: Invalid response from API: Timeout occurred while connecting to IPP server.
2020-06-06 10:36:51 ERROR (MainThread) [homeassistant.components.ipp] Error fetching ipp data: Invalid response from API: Timeout occurred while connecting to IPP server.
2020-06-06 10:45:28 ERROR (MainThread) [homeassistant.components.ipp] Error fetching ipp data: Invalid response from API: Timeout occurred while connecting to IPP server.
2020-06-06 10:53:49 ERROR (MainThread) [homeassistant.components.ipp] Error fetching ipp data: Invalid response from API: Timeout occurred while connecting to IPP server.
2020-06-06 10:55:59 ERROR (MainThread) [homeassistant.components.ipp] Error fetching ipp data: Invalid response from API: Timeout occurred while connecting to IPP server.
2020-06-06 11:03:15 ERROR (MainThread) [homeassistant.components.ipp] Error fetching ipp data: Invalid response from API: Timeout occurred while connecting to IPP server.
2020-06-06 11:33:08 ERROR (MainThread) [homeassistant.components.ipp] Error fetching ipp data: Invalid response from API: Timeout occurred while connecting to IPP server.
2020-06-06 11:55:44 ERROR (MainThread) [homeassistant.components.ipp] Error fetching ipp data: Invalid response from API: Timeout occurred while connecting to IPP server.
2020-06-06 12:23:30 ERROR (MainThread) [homeassistant.components.ipp] Error fetching ipp data: Invalid response from API: Timeout occurred while connecting to IPP server.
2020-06-06 12:38:54 ERROR (MainThread) [homeassistant.components.ipp] Error fetching ipp data: Invalid response from API: Timeout occurred while connecting to IPP server.
2020-06-06 12:57:36 ERROR (MainThread) [homeassistant.components.ipp] Error fetching ipp data: Invalid response from API: Timeout occurred while connecting to IPP server.
2020-06-06 14:03:36 ERROR (MainThread) [homeassistant.components.ipp] Error fetching ipp data: Invalid response from API: Timeout occurred while connecting to IPP server.
2020-06-06 14:53:38 ERROR (SyncWorker_3) [homeassistant.util.yaml.loader] invalid key: "OrderedDict([('mdi:wifi-strength-alert-outline', None)])"
  in "/home/homeassistant/.homeassistant/configuration.yaml", line 156, column 0
2020-06-06 21:40:40 WARNING (MainThread) [homeassistant.components.websocket_api.http.connection.2675533040] Disconnected: Did not receive auth message within 10 seconds
2020-06-06 22:03:57 ERROR (SyncWorker_0) [homeassistant.components.recorder.util] Error executing query: (sqlite3.OperationalError) database is locked
(Background on this error at: http://sqlalche.me/e/e3q8)
